src/support/check/checkContainsText.js

```diff
@@ -36,9 +36,9 @@ export default (elementType, selector, falseCase, expectedText) => {
      * The text of the element
      * @type {String}
      */
-    const elem = $(selector);
+    const elem = await $(selector);
```

Comment:
I've seen this a couple of times... Is this requested by the framework? I mean, are the selectors supposed to be asynchronous? Or are just the methods called on the resulting `elem` from the selector the ones that are async?